catalog/pipeline/fbc/0.1/README.md

```diff
@@ -0,0 +1,13 @@
+# Milestone-9 FBC Pipeline
+
+FBC Pipeline
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| applicationSnapshot | The ApplicationSnapshot in JSON format | No | - |
```

Comment:
application prefixes have been removed no? We should alter the parameters accordingly

Reply:
+1